test/opt/pass_remove_duplicates_test.cpp

```diff
@@ -307,6 +307,9 @@ OpGroupDecorate %2 %4
   EXPECT_EQ(GetErrorMessage(), "");
 }
 
+// Test what happens when a type is a resource type.  For now we are merging
```

Comment:
If you think its easier, consider just disabling the older version of the test.

Reply:
I had thought of that, but I believe the test could get lost that way.  This way if some tries to do something similar to what I did, this test will start to fail, and make it obvious which cases they need to handle.